lib/diagnostic.js

```diff
@@ -47,7 +47,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire().bind(alderaan);
```

Comment:
Close! 
In order for this to work, you have to call the bound function: 
`deathStar.fire().bind(alderaan)()`

Or use `.call()`, which calls it automatically.
`deathStar.fire().call(alderaan)`

Also, in order to pass tests, you need to return it: `return deathStar.fire().call(alderaan)`